test/csp-parser.test.js

```diff
@@ -1,6 +1,6 @@
 import { assert } from "chai";
 
-import { parseCsp } from "../src/analyzer/cspParser.js";
+import { DUPLICATE_WARNINGS_KEY, parseCsp } from "../src/analyzer/cspParser.js";
```

Comment:
Nit: I would recommend against importing constants in the tests, as this means changing the constant value does not require updating the test.